index.js

```diff
@@ -166,9 +178,10 @@ class ExecutorQueue extends Executor {
      * @param {Object} config.pipeline Pipeline of the job
      * @param {Object} config.job      Job object to create periodic builds for
      * @param {String} config.apiUri   Base URL of the Screwdriver API
+     * @param {String} config.eventId  Parent event ID (optional)
```

Comment:
can put `[config.eventId]` to annotation that it is optional
Also, type should be Number

Reply:
Fixed